### DISCUSSION TITLE
Build Slo Generator from bigquery input

### DISCUSSION THREAD

**Discussion by minawad** [google/slo-generator#314] [Ideas]
We need to use SLO Generator from data stored in bigquery, could you add this feature please.

**Comment by lvaylet** [comment]
Hi @minawad, and thanks for posting!

The "Discussions" space in GitHub is best suited for general questions and answers. For feature requests like this one, the best option is the "Issues" space, specifically the ["Feature Request"](https://github.com/google/slo-generator/issues/new/choose) template.

Would you mind filling the template so we understand your requirements better, providing as many details and sample use cases as possible?

In addition, we can have a 30-min meeting as discussed during the Art of SLOs workshop last week.

**Reply by minawad** [reply]
Hi @lvaylet ,

Feature request is created https://github.com/google/slo-generator/issues/317
I'm available today between 3-4pm, if are avalaible let me know. if not could you provde me your availability please

Regards,
Minas
